components/common-data-items.js

```diff
@@ -115,7 +115,12 @@ DonorDataItems.commonProperties = [
 /**
  * Display data items common to all sample-derived objects.
  */
-export function SampleDataItems({ item, sources = null, children }) {
+export function SampleDataItems({
+  item,
+  sources = null,
+  children,
+  sortedFractions,
```

Comment:
Always put `children` last because it gets treated differently from other props.

Reply:
done